mlos_bench/mlos_bench/services/types/fileshare_type.py

```diff
@@ -31,12 +33,14 @@ def download(self, remote_path: str, local_path: str, recursive: bool = True) ->
             if True (the default), download the entire directory tree.
         """
 
-    def upload(self, local_path: str, remote_path: str, recursive: bool = True) -> None:
+    def upload(self, params: dict, local_path: str, remote_path: str, recursive: bool = True) -> None:
```

Comment:
Required for ssh fileshare service.
- [ ] Might want to break this out as well.

Reply:
Moved to #517